source/material_model/melt_simple.cc

```diff
@@ -0,0 +1,699 @@
+          depletion_density_change   = prm.get_double ("Depletion density change");
+          depletion_solidus_change   = prm.get_double ("Depletion solidus change");
+
+          if (thermal_viscosity_exponent!=0.0 && reference_T == 0.0)
+            AssertThrow(false, ExcMessage("Error: Material model Melt simple with Thermal viscosity exponent can not have reference_T=0."));
+
+          A1              = prm.get_double ("A1");
+          A2              = prm.get_double ("A2");
+          A3              = prm.get_double ("A3");
+          B1              = prm.get_double ("B1");
+          B2              = prm.get_double ("B2");
+          B3              = prm.get_double ("B3");
+          C1              = prm.get_double ("C1");
+          C2              = prm.get_double ("C2");
+          C3              = prm.get_double ("C3");
+          r1              = prm.get_double ("r1");
+          r2              = prm.get_double ("r2");
+          beta            = prm.get_double ("beta");
+          peridotite_melting_entropy_change
+            = prm.get_double ("Peridotite melting entropy change");
+          M_cpx           = prm.get_double ("Mass fraction cpx");
+        }
+        prm.leave_subsection();
+      }
+      prm.leave_subsection();
+    }
+  }
+}
+
+// explicit instantiations
+namespace aspect
+{
+  namespace MaterialModel
+  {
+    ASPECT_REGISTER_MATERIAL_MODEL(MeltSimple,
+                                   "melt simple",
+                                   "A material model that implements a simple formulation of the "
+                                   "material parameters required for the modelling of melt transport, "
+                                   "including a source term for the porosity according to the melting "
+                                   "model for dry peridotite of Katz, 2003. This also includes a "
+                                   "computation of the latent heat of melting (if the latent heat "
+                                   "heating model is active)."
+                                   "\n"
+                                   "Most of the material properties are constant, except for the shear, "
+                                   "viscosity $\\eta$, the compaction viscosity $\\xi$, and the "
+                                   "permeability $k$, which depend on the porosity; and the solid and melt "
+                                   "densities, which depend on temperature and pressure:\n "
+                                   "$\\eta(\phi,T) = \\eta_0 e^{\\alpha(\\phi-\\phi_0)} e^{-\\beta(T-T_0)/T_0}$,\n"
+                                   "$\\xi(\\phi,T) = \\xi_0 \frac{\\phi_0}{\\phi} e^{-\\beta(T-T_0)/T_0}$,\n"
+                                   "$k=k_0 \\phi^n (1-\\phi)^m$,\n"
+                                   "$\\rho=\\rho_0 (1 - \\alpha (T - T_\\text{adi})) e^{\\kappa p}$."
+                                   "\n"
```

Comment:
you need *two* `\n` to get a paragraph break (the first just ends the line, the second produces an empty line).